content/_data/logo/lausanne.yml

```diff
@@ -0,0 +1,8 @@
+---
+name: 'Lausanne'
```

Comment:
It is a Swiss Jenkins Meetup, not a Lausanne one

Reply:
updated it to switzerland.